app/src/main/java/edu/uc/forbesne/shoppingsidekick/service/ProductService.kt

```diff
@@ -22,7 +22,7 @@ class ProductService {
         return MutableLiveData<ArrayList<Product>>()
     }
 
-    fun fetchAllProductsFromOneStore() : MutableLiveData<ArrayList<Product>> {
+    internal fun fetchAllProductsFromOneStore() : MutableLiveData<ArrayList<Product>> {
```

Comment:
This is duplicated in an earlier code review, in which case... first one in wins.